test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/ViewFeatures/DefaultHtmlGeneratorTest.cs

```diff
@@ -190,6 +191,104 @@ public void GenerateTextArea_WithNullExpression_WithNameAttribute_DoesNotThrow()
             Assert.Equal(expected, attribute.Value);
         }
 
+        [Fact]
+        public void GenerateTextArea_RendersMaxLength_WhenMaxLengthAttributePresent()
```

Comment:
Would be fine for this and `GenerateTextArea_RendersMaxLength_WhenStringLengthAttributePresent()` to be a single `[Theory]`. The parameters would be `string expression` and `int expectedValue`.